examples/kinematics/simple_harmonic_oscillations_of_spring.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+from sympy import solve, symbols, Eq
+from symplyphysics import Quantity, units, print_expression, convert_to
+from symplyphysics.laws.dynamics import (
```

Comment:
It uses laws from `dynamics` so probably should be in `dynamics` as well.

Reply:
Hm, that's sensible, I'll move it

Reply:
Sync with repository before doing it. I've added some improvements.